app/main.ipynb

```diff
@@ -671,23 +952,34 @@
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T14:20:27.503470Z",
-     "start_time": "2023-07-03T14:20:27.458680Z"
+     "end_time": "2024-04-25T09:35:41.960350Z",
+     "start_time": "2024-04-25T09:35:41.953665Z"
     }
    }
   },
   {
    "cell_type": "code",
-   "execution_count": 35,
-   "outputs": [],
+   "execution_count": 68,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "array([ 3, 13, 11, 24, 21])"
+     },
+     "execution_count": 68,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your numpy code here"
+    "my_array = np.array([6, 9, 12, 14, 17, 20])\n",
+    "mask = (my_array % 2 == 0) & (my_array % 3 == 0) | (my_array > 10)\n",
+    "np.bitwise_xor(my_array[mask], 7) + 2"
```

Comment:
incorrect result
the issue is with the order of execution binary operations
original: if num % 2 == 0 and (num % 3 == 0 or num > 10)
if you remove parenthesises(this is what you did), it'll look like
if num % 2 == 0 and num % 3 == 0 or num > 10
the execution ordering of such operators is: not, and, or
in your case and will be executed first with num % 2 == 0 and num % 3 == 0 and then or condition
if you add parenthesises, then "if num % 2 == 0 and (num % 3 == 0 or num > 10)"
"and" will be executed between the first condition and the result of "or" operation, so parenthesises matter!